### PR TITLE
virttest.defaults: Added support for default machine type based on arch

### PR DESCRIPTION
**Pull request by sathnaga**
Added support for choosing default machine type
based on machine architecture

Addresses https://github.com/avocado-framework/avocado-vt/issues/420

### CODE REVIEW

**Comment by will-Do** [comment]
LGTM

File: virttest/defaults.py [hunk 1]
```diff
@@ -1,9 +1,22 @@
+import platform
+ARCH = platform.machine()
+# Set the default machine to i440fx for unidentifed arch also
+# Below line can be removed once support for all possible arch is covered
 DEFAULT_MACHINE_TYPE = "i440fx"
 
+if ARCH in ('ppc64', 'ppc64le'):
+    DEFAULT_MACHINE_TYPE = "pseries"
+elif ARCH in ('x86_64', 'i386'):
+    DEFAULT_MACHINE_TYPE = "i440fx"
```
Comment (xutian) [inline virttest/defaults.py]: Q35 machine type maybe the default machine type in future.

Comment (xutian) [inline virttest/defaults.py]: how about to set it to PC?

Comment (sathnaga) [inline virttest/defaults.py]: @will-Do Thanks :-)
@xutian : Seeting it to PC might not be common for all, we can have Q35 aswell, This is for enabling the arch dependent defaults, we can decide/change values later aswell.

**Comment by xutian** [comment]
Ok,  Acked-by: Xu Tian <xutian@redhat.com>

**Comment by xutian** [comment]
Merged, thanks @sathnaga and @will-Do

**Pull request merged**